sklearn/svm/tests/test_svm.py

```diff
@@ -1425,17 +1424,18 @@ def test_n_support_oneclass_svr():
     # this is a non regression test for issue #14774
     X = np.array([[0], [0.44], [0.45], [0.46], [1]])
     clf = svm.OneClassSVM()
-    assert not hasattr(clf, "n_support_")
-    clf.fit(X)
-    assert clf.n_support_ == clf.support_vectors_.shape[0]
-    assert clf.n_support_.size == 1
-    assert clf.n_support_ == 3
+    with pytest.warns(FutureWarning):
+        assert not hasattr(clf, "n_support_")
+        clf.fit(X)
+        assert clf.n_support_ == clf.support_vectors_.shape[0]
+        assert clf.n_support_.size == 1
+        assert clf.n_support_ == 3
```

Comment:
Interesting that we test this. I would be okay with testing `_n_support[0]` directly for these properties.